src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -64,7 +64,7 @@ function renderButton(sc, action, display, style, showText, asItem, autoFocus, t
         size={48}
         isDisabled={isDisabled}
         onClick={action}
-        ariaLabel={hint || text}
+        ariaLabel={text}
```

Comment:
What were the hints doing earlier. Since we are removing hint, do we know what else gets affected?

Reply:
Let's resolve this after @adhmenon replies.

Reply:
Thanks for the tip Parv. Yes, it works even with all buttons. I checked the older behaviour and it is the same for all buttons (except ofc the join meeting one). 
I think they were not setting hint for most buttons and for join meeting they were but this causes an issue in the voiceover. I think it has not broken anything else.
Here is a [Vidcast]() showing the proof.